### PR TITLE
Exams by Dmytro Mazurenko IM-33

### PR DESCRIPTION
**Pull request by gbdfxilhn**

### CODE REVIEW

**Comment by alomj** [comment]
you have done a great job!

**Comment by DavidkJun** [comment]
I can agree with all changes after the review

File: Tasks/between.js [hunk 1]
```diff
@@ -1,21 +1,18 @@
 // Extract substring between prefix and suffix
 
-getvaluebetween = (str, p, s) => {
-  i = str.indexOf(p);
-  if (i === -1) return '';
+const getValueBetween = (fullString, prefix, suffix) => {
+  const index = fullString.indexOf(prefix);
+  if (index === -1) return '';  
   else {
```
Comment (tshemsedinov) [inline Tasks/between.js:6]: Do not need else here

File: Tasks/between.js [hunk 2]
```diff
@@ -1,21 +1,18 @@
...
+    let prefixLength = index + prefix.length;
+    fullString = fullString.substring(prefixLength);
+    if (suffix) {
+      index = fullString.indexOf(suffix);
+      if (index === -1)  return '';
+      else fullString = fullString.substring(0, index);
     }
   }
-  return str;
+  return fullString;
```
Comment (tshemsedinov) [inline Tasks/between.js:15]: Do not modify incoming args

**Review by tshemsedinov — COMMENTED** [review]
Need eslint and prettier to format and check code style

**Pull request closed without merging**